### DISCUSSION TITLE
Pay button warning about commercial use

### DISCUSSION THREAD

**Discussion by veebch** [btcpayserver/btcpayserver#3534] [Ideas / Feature Requests]
Loving your work! One minor point:

The current warning text regarding commercial use of Play Buttons is not clear, in that it does not describe the potential vulnerability with clarity. Perhaps insert a link to a page that gives a little more detail. The [FAQ on pay buttons](https://docs.btcpayserver.org/FAQ/Apps/#what-is-a-payment-button) seems to need updating anyway.

**Comment by pavlenex** [comment]
This has been completed a while ago, so closing this discussion.  In case it's still ambiguous, feel free to let me know which part and we can work on the copy.
 
<img width="820" alt="Screenshot 2025-05-06 at 11 43 26" src="https://github.com/user-attachments/assets/e2c82a91-4ffc-4abb-bd4b-a52a456e830c" />
